scripts/generate-bootstraps.sh

```diff
@@ -13,6 +13,11 @@ trap 'rm -rf $BOOTSTRAP_TMPDIR' EXIT
 # and <10.
 BOOTSTRAP_ANDROID10_COMPATIBLE=false
 
+# Optional caching of downloaded packages in output directory
+# Apt only for now
+TERMUX_PACKAGE_DOWNLOAD_CACHE=false
+TERMUX_PACKAGE_DOWNLOAD_CACHE_DIR="$PWD/output"
```

Comment:
This should be named `cache` instead of `output`.

 The `TERMUX_PACKAGE_DOWNLOAD_CACHE_DIR` does not include `TERMUX_APP_PACKAGE`, which will result in mixing of packages with different prefixes if cache dir is not cleaned manually if repo url is changed.
 
 Moreover, the `generate-bootstrap.sh` script is not checking if `TERMUX_REPO_PACKAGE` equals `TERMUX_APP_PACKAGE`. It would be better to replace `REPO_BASE_URLS` and `-r` format to `<package_name>:<repo_url>` so that caching is correctly done as you want.
 
 https://github.com/termux/termux-packages/blob/8e8b1a40ab8c05e20f2f1c3c90c0abecf780f3a0/scripts/properties.sh#L35-L38